prov/rxm/src/rxm.h

```diff
@@ -86,6 +86,11 @@
 	       " (fi_addr: 0x%" PRIx64 " tag: 0x%" PRIx64 ")\n",\
 	       addr, tag)
 
+#define RXM_GET_PROTO_STATE(comp)			\
+	((enum rxm_proto_state *)			\
+	 (((unsigned char *)comp->op_context +		\
+		offsetof(struct rxm_buf, state))))
```

Comment:
Can we update this to return the state, rather than a pointer to the state?

Reply:
Sure, no problem.